libcontainer/seccomp/types.go

```diff
@@ -0,0 +1,48 @@
+package seccomp
```

Comment:
Given this, I'd be willing move to the existing API, but I'd like to make some of these changes at the same time.